shell/platform/windows/cursor_handler.cc

```diff
@@ -45,6 +46,31 @@ void CursorHandler::HandleMethodCall(
     const auto& kind = std::get<std::string>(kind_iter->second);
     delegate_->UpdateFlutterCursor(kind);
     result->Success();
+  } else if (method.compare(kSetSystemCursorMethod) == 0) {
+    const auto& map = std::get<EncodableMap>(*method_call.arguments());
+    auto buffer = std::get<std::vector<uint8_t>>(
+        map.at(flutter::EncodableValue("buffer")));
+    auto width = std::get<int>(map.at(flutter::EncodableValue("width")));
+    auto height = std::get<int>(map.at(flutter::EncodableValue("height")));
+    auto hotx = std::get<double>(map.at(flutter::EncodableValue("hotx")));
+    auto hoty = std::get<double>(map.at(flutter::EncodableValue("hoty")));
+    HCURSOR cursor = nullptr;
+    // Flutter returns rawRgba, which has 8bits*4channels
+    auto bitmap = CreateBitmap(width, height, 1, 32, &buffer[0]);
+    if (bitmap == nullptr) {
+      result->Error("Argument error", "Invalid rawRgba bitmap from flutter");
+      return;
+    }
+    ICONINFO icon_info;
+    icon_info.fIcon = 0;
+    icon_info.xHotspot = hotx;
+    icon_info.yHotspot = hoty;
+    icon_info.hbmMask = bitmap;
+    icon_info.hbmColor = bitmap;
+    cursor = CreateIconIndirect(&icon_info);
```

Comment:
It looks like [this can fail and return `NULL`](https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-createiconindirect#return-value). Can we check this and return an error result?

Reply:
yes, i agree with this. a check is added in L108